standalone/runtime/tests/test_issue.rs

```diff
@@ -667,101 +682,110 @@ mod execute_pending_issue_tests {
     #[test]
     fn integration_test_issue_execute_postcond_exact_payment() {
         test_with_initialized_vault(|vault_id| {
-            let requested_btc = vault_id.wrapped(1000);
-            let (issue_id, issue) = request_issue(&vault_id, requested_btc);
-            let post_request_state = ParachainState::get(&vault_id);
-
-            ExecuteIssueBuilder::new(issue_id).assert_execute();
-
-            // user balances are updated, tokens are minted and fees paid
-            assert_eq!(
-                ParachainState::get(&vault_id),
-                post_request_state.with_changes(|user, vault, _, fee_pool| {
-                    (*user.balances.get_mut(&DEFAULT_GRIEFING_CURRENCY).unwrap()).locked -= issue.griefing_collateral();
-                    (*user.balances.get_mut(&DEFAULT_GRIEFING_CURRENCY).unwrap()).free += issue.griefing_collateral();
-                    (*user.balances.get_mut(&vault_id.wrapped_currency()).unwrap()).free += issue.amount();
-
-                    *fee_pool.rewards_for(&vault_id) += issue.fee();
-                    vault.issued += requested_btc;
-                    vault.to_be_issued -= requested_btc;
-                })
-            );
-
-            // issue request is updated: status is complete
-            assert_eq!(
-                IssuePallet::issue_requests(issue_id).unwrap().status,
-                IssueRequestStatus::Completed
-            );
+            for griefing_currency in [DEFAULT_GRIEFING_CURRENCY, vault_id.collateral_currency()] {
```

Comment:
According to the implementation, the griefing currency could be set to anything, not just vault collateral currencies. Not saying this test is wrong but it seems to assume that the griefing currency will not be e.g. `IBTC`

Reply:
It's main purpose was to test in the default griefing collateral plus one other one, it doesn't really matter which one it is. But maybe it makes more sense to use a hardcoded one